### ISSUE TITLE
conflict with bootstrap

### ISSUE THREAD

**Issue by kefengwei** [vmware/clarity#2960]
Because i need use bootstrap's utilities, just like spacing,display. So i import bootstrap's into project. But there have some conflict , just like dropdown-toggle. Can clarity appropriate work with bootstrap now?

**Comment by gnomeontherun** [comment]
Hi there, this is really a question for StackOverflow since its kind of an implementation question.

We have some utilities included from Bootstrap, but we are locked on version 4.0.0-alpha.5. If you import other versions of Bootstrap, there might be conflicts depending on which one is loaded first. You'd have to review the changelogs carefully to see if any of the utilities you import have breaking changes and could cause issues for Clarity, there are a few which is why we haven't updated. Our goal is to eventually decouple from Bootstrap, but that is a long term vision.

In short, either use Bootstrap 4.0.0-alpha.5, or create your own utilities for spacing and display.
